package.json

```diff
@@ -54,7 +54,7 @@
     "renameschema": "scripts/rename-schema.sh",
     "presite": "yarn data && yarn schema && yarn build:site && yarn build:versions && scripts/create-example-pages.sh",
     "site": "yarn site:only",
-    "site:only": "pushd site && bundle exec jekyll serve -I -l && popd",
+    "site:only": "pushd site && bundle exec jekyll serve -I -l -H 0.0.0.0 && popd",
```

Comment:
Leave the default port.

Reply:
It was needed to connect to the site running in docker from  "outside" (= from host).
Default is "localhost", which does not work.
Not sure, if running in VScode" needs it. Will check.

Reply:
not needed in VScode